### PR TITLE
Remove supportsRtl from AndroidManifest.xml

### PR DESCRIPTION
**Pull request by SUPERCILEX**
Speeds up merging from https://github.com/firebase/FirebaseUI-Android/pull/529#issuecomment-273586245 discussion while still giving credit to original author.

### CODE REVIEW

File: library/quality/lint-baseline.xml [hunk 1]
```diff
@@ -2,13 +2,24 @@
...
+
+    <issue
         id="UnusedIds"
         message="The resource `R.id.facebook_button` appears to be unused"
         errorLine1="    android:id=&quot;@+id/facebook_button&quot;/>
 "
         errorLine2="    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
         <location
-            file="src\main\res\layout\idp_button_facebook.xml"
+            file="src/main/res/layout/idp_button_facebook.xml"
```
Comment (SUPERCILEX) [inline library/quality/lint-baseline.xml]: Dunno why we had windows style directories.

**Comment by samtstern** [comment]
@SUPERCILEX thanks for this.  I would normally squash and merge this but, as you said, credit to the original author is one of the goals here.  Could you take your 4 commits and rebase them into 1 so that I can merge this directly but only have one commit from each of you?

**Comment by SUPERCILEX** [comment]
@samtstern YESSSS! Thank you so much! This was my first experience rebasing anything and it's pretty cool! (and scary since you can destroy history!) Thanks for forcing me to learn this, I love FirebaseUI! 😄

**Comment by samtstern** [comment]
Hah now you know the terrifying power of `git rebase`.  I will merge this and close #529 once the build passes.

**Comment by samtstern** [comment]
Oooh actually it looks like @ay1man4 never signed the CLA, so we actually can't merge his commit :-1:

**Comment by SUPERCILEX** [comment]
@samtstern Dang, should I just remove his commit?

**Comment by samtstern** [comment]
@SUPERCILEX ok given the lack of recent responses from @ay1man4 let's just rebase this all so it comes from you and merge it.

We can give @ay1man4 some other opportunities to contribute in the future.

**Comment by SUPERCILEX** [comment]
@samtstern Just found out @ay1man4 signed the CLA: https://github.com/firebase/FirebaseUI-Android/pull/529#issuecomment-274956893.

**Pull request merged**